.github/workflows/build.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v3
         with:
-          node-version: 17.x
+          node-version: 18.x
```

Comment:
should we just say 18.16.1 since we're using a specific version everywhere else?

Reply:
i'm wondering if we should say 18.x everywhere else instead since the long term support version can update occasionally -- my list from `nvm ls` lists `lts/hydrogen` as 18.17.0

<img width="333" alt="Screen Shot 2023-08-08 at 9 11 35 PM" src="https://github.com/tally-team/tally-frontend/assets/24977851/e4b1b97a-fc27-4db0-b7d4-bb73f323f7cd">

further down in the PR has `>=18.16.1` listed in the engine properties so we should be consistent whether we want to be strictly on one version of node 18 or let it be 18.x

Reply:
If we say 18.x, won't it just go with the most recent which can conflict with current dependencies? Unless the fixed dependency is deprecated, I think we should pin the dependency to a specific version. If it gets deprecated, we can then update.

Reply:
I'm not tied to any particular version of node 18, but yes, I agree, we may as well pin the version entirely. I'll update it to 18.17.0.